geocode_lat_long/geocode_lat_long.md

```diff
@@ -0,0 +1,263 @@
+<!--
+
+author:   Elizabeth Drellich
+email:    drelliche@chop.edu
+version:  1.0.0
+module_template_version: 3.0.0
+language: en
+narrator: UK English Female
+title: Encoding Geospaital Data: Latitude and Longitude
+comment:  This is an introduction to latitude and longitude and the importance of geocoding - encoding geopspatial data in this coordinate system.
+long_description: If you use any geospatial data, such as patient or participant addresses, it is important that that location data be in a usable form. This means using the same coordinate system that Global Positioning Systems use: latitude and longitude. This module is appropriate, as either an introduction or review, for anyone considering using geospatial data in their analysis. 
+estimated_time: 15 minutes
+
+@learning_objectives  
+
+After completion of this module, learners will be able to:
+
+- understand the importance of geocoding addresses
+- understand the latitude and longitude coordinate system
```

Comment:
```suggestion
- Understand the latitude and longitude coordinate system
```
You know I am funny about starting things with a capital letter, but you do you, I'm willing to be overridden...

Reply:
I'm capital letter flexible :)